### PR TITLE
added more name and value options for /embed

### PR DESCRIPTION
**Pull request by Timmie644**

### CODE REVIEW

**Review by codeman1o1 — APPROVED** [review]

**Comment by Timmie644** [comment]
indented code @codeman1o1

File: slash/developer.py [hunk 1]
```diff
@@ -52,6 +52,85 @@ async def embed(
...
+        valu7: str = SlashOption(
+            name="value7", description="The value of the 7th field", required=False
+        ),
+        name8: str = SlashOption(
+            name="name8", description="The name of the 8th field", required=False
+        ),
+        value8: str = SlashOption(
+            name="value8", description="The value of the 8th field", required=False
+        ),
+        name8: str = SlashOption(
```
Comment (codeman1o1) [inline slash/developer.py:85]: Please remove duplicate paramaters

File: slash/developer.py [hunk 2]
```diff
@@ -52,6 +52,85 @@ async def embed(
...
+        name6: str = SlashOption(
+            name="name6", description="The name of the 6th field", required=False
+        ),
+        value6: str = SlashOption(
+            name="value6", description="The value of the 6th field", required=False
+        ),
+        name7: str = SlashOption(
+            name="name7", description="The name of the 7th field", required=False
+        ),
+        valu7: str = SlashOption(
```
Comment (codeman1o1) [inline slash/developer.py:76]: Please fix this typo

File: slash/developer.py [hunk 3]
```diff
@@ -52,6 +52,85 @@ async def embed(
...
+        value14: str = SlashOption(
+            name="value14", description="The value of the 14th field", required=False
+        ),
+        name15: str = SlashOption(
+            name="name15", description="The name of the 15th field", required=False
+        ),
+        value15: str = SlashOption(
+            name="value15", description="The value of the 15th field", required=False
+        ),
+    ):
```
Comment (codeman1o1) [inline slash/developer.py:133]: Remove double ending brackets

File: slash/developer.py [hunk 4]
```diff
@@ -77,6 +156,79 @@ async def embed(
                     name=name3,
                     value=value3,
                     inline=False,
+                )
+                                )
```
Comment (codeman1o1) [inline slash/developer.py:160]: Remove this line

**Review by codeman1o1 — CHANGES_REQUESTED** [review]

**Comment by Timmie644** [comment]
fixed the thing @codeman1o1

**Comment by codeman1o1** [comment]
You did not fix [this](https://github.com/codeman1o1/TIJK-Bot/pull/121#discussion_r836583344)

**Comment by Timmie644** [comment]
now I did fix it

**Pull request merged**